https://bugs.gentoo.org/830325
https://gitlab.freedesktop.org/realmd/adcli/-/merge_requests/47

From: Henning Schild <henning@hennsch.de>
Date: Tue, 11 Jan 2022 08:22:55 +0100
Subject: [PATCH] library: include endian.h for le32toh

upstream commit 938065a751c0876eb837a27f8c1443fc7d0d2551

linking against musl libc we need to include that specifically, actually
also according to "man 3 endian".
--- a/library/adutil.c
+++ b/library/adutil.c
@@ -29,6 +29,7 @@
 
 #include <assert.h>
 #include <ctype.h>
+#include <endian.h>
 #include <errno.h>
 #include <stdio.h>
 #include <stdlib.h>
